### ISSUE TITLE
[NuGet.org Bug]: 

### ISSUE THREAD

**Issue by dauthleikr** [NuGet/NuGetGallery#9534]
### Impact

I'm unable to use NuGet.org

### Describe the bug

I am not able to upload a generated snupkg to Nuget.org. During indexing it always reports this error "The uploaded symbols package contains one or more pdbs that are not portable."

The snupkg in question contains a .NET assembly with a portable PDB (file starts with "BSJB"), but also a C++ component and its pdb. I suspect the portability-validation fails for the C++ pdb (starting with `Microsoft C/C++ MSF 7.00`), but there is nothing I can do here, since as far as I know, the portable-pdb format is a concept for .NET assemblies only.

### Repro Steps

1. Ship snupkg with valid pdb from .NET assembly, and PDB from generated by MSVC in contentFiles
2. Try uploading to nuget.org

### Expected Behavior

Do not validate C++ pdbs

**Comment by dauthleikr** [comment]
Is the correct way here to just not ship the C++ pdb with the snupkg, and only deliver it via contentFiles in the nupkg?

**Comment by joelverhagen** [comment]
I think your theory about it failing on the C++ PDB. The NuGet.org symbol server only accepts so-called "portable" PDBs. We can't change this requirement for security reasons. However, excluding that particular PDB sounds like a setting you would need to put in the project file (some kind of setting for NuGet client code that does `pack` to read).

I'll pull in some NuGet client folks to see if there's a way to exclude some PDBs (`Grind.LowLevel.pdb`) from the .snupkg. Frankly, it's strange to me that content files are included in the PDB at all. I would expect just .pdbs in runtime top level directories like `lib`.

**Comment by dauthleikr** [comment]
@joelverhagen Yes, excluding PDB's is what I'd need to make a clean package. I guess even changing the default behavior so that only PDB's are packed, which can be used by the NuGet symbol server, could also be sensible.
